mapeditor/mapcontroller.cpp

```diff
@@ -381,13 +381,14 @@ void MapController::pasteFromClipboard(int level)
 	if(_clipboardShiftIndex == int3::getDirs().size())
 		_clipboardShiftIndex = 0;
 	
+	QStringList errors;
+	QString errorMsg;
```

Comment:
not sure why declaring outside the loop

Reply:
Wanted to use it again with `std::vector` approach, but then changed to `QStringList` which has this nice `join()` function. Should be back in loop or dosn't matter ?

Reply:
I mean, only errorMsg should be inside the loop, makes no sense to have it outside